server/server.go

```diff
@@ -189,23 +203,79 @@ func (s *audioHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *stickerHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	messageIDs, ok := r.URL.Query()["messageId"]
+	if !ok || len(messageIDs) == 0 {
+		s.logger.Error("no messageID")
+		return
+	}
+	messageID := messageIDs[0]
+	var stickerHash string
+	err := s.db.QueryRow(`SELECT sticker_hash FROM user_messages WHERE id = ?`, messageID).Scan(&stickerHash)
```

Comment:
i think there is no need to "hide" hash over messageid for stickers, we could add hash right in the sticker URL, so in that case we don't need to query it from db